public/templates/public/sidebar.html

```diff
@@ -5,13 +5,34 @@ <h4 class="font-italic">{{ welcome.welcome_title }}</h4>
 
 
 <div class="card p-4">
-  <h4 class="font-bold">Nächstes Training</h4>
-  <p class="mb-0">{{next_training.date}}, {{next_training.location}}</p>
-  {% if partn > 0 %}
-  <p class="mb-0">mindestens {{partn}} Person kommt</p>
-  {% elif partn > 1 %}
-  <p class="mb-0">mindestens {{partn}} Personen kommen</p>
-  {%endif%}
+    <h4 class="font-bold">Nächstes Training</h4>
+    <p class="mb-0">{{next_training.date}}, {{next_training.location}}</p>
+    {% if user.is_authenticated %}
+    <div class="row">
+        <div class="col">
+            <small>who's in? 👍</small>
+        </div>
+    </div>    
+    <div class="container p-0 m-0">
+
+        <div class="row float-left pl-0 ml-0" style="line-height:1;"> 
+            {% for part in coming %}
+            <div class="col m-0 p-0">
+                <img src="{{part.person.profile.image_url}}" 
+                class="img m-0 p-0" 
+                data-toggle="tooltip" 
+                data-placement="top"
+            title="{{part.person.first_name}}"
```

Comment:
ah, und hier ist ein title-attribut, das oben in der eventslist nicht auftaucht. noch ein argument für die modularisierung ;)